plugins/flytekit-memray/flytekitplugins/memray/profiling.py

```diff
@@ -0,0 +1,74 @@
+import os
+import time
+from typing import Callable, List, Optional
+
+import memray
+from flytekit import Deck
+from flytekit.core.utils import ClassDecorator
+
+
+class memray_profiling(ClassDecorator):
+    def __init__(
+        self,
+        task_function: Optional[Callable] = None,
```

Comment:
Can we allow for `native_traces`, `trace_python_allocators`, `follow_fork`, `memory_interval_ms` from `memray.Tracker` to be passed through here?

With `trace_python_allocators=True`, I do not think we need to set `PYTHONMALLOC` in the environment anymore.

Reply:
Nice catch! Added these 👍